ctrd/utils_image_proxy.go

```diff
@@ -0,0 +1,155 @@
+// HTTP related code copy from src/net/http/transport.go
+
+package ctrd
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+)
+
+var proxy string
```

Comment:
Could you move this field to https://github.com/alibaba/pouch/blob/master/ctrd/client.go#L21-L32 ?

Will it be better rather than a local variable in package ctrd. @Ace-Tang

Reply:
I think proxy do not need to add to  Client Struct, since they have no such strong relation.
And proxy only set when daemon start. WDYT, @allencloud